striatum/bandit/exp3.py

```diff
@@ -102,9 +101,8 @@ def get_action(self, context):
         else:
             action_max = six.next(self.exp3_)
 
-        self.last_history_id += 1
-        self._historystorage.add_history(np.transpose(np.array([context])), action_max, reward=None)
-        return self.last_history_id, action_max
+        last_history_id = self._historystorage.add_history(np.transpose(np.array([context])), action_max, reward=None)
```

Comment:
`last_history_id` -> `history_id` would be better. You don't actually have a "last" context here.